v1/openapi/schemas/child_tree_request.yaml

```diff
@@ -0,0 +1,69 @@
+x-component: "schemas/childTree"
+description: Request representation.
+allOf:
+  - $ref: 'record-abstract.yaml'
+  - type: object
+    properties:
```

Comment:
We also have property `root` in docs for this request

Reply:
Add field root